.github/workflows/linux.yml

```diff
@@ -0,0 +1,186 @@
+            URL: LINUX_CUBIT_2020_2
+            URL_SDK: LINUX_CUBIT_SDK_2020_2
+
+    name: 'Trelis/Cubit Svalinn Plugin ${{ matrix.cubit }} Build for Linux'
+
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Environment Variables
+        run: |
+          echo "CURRENT=$(pwd)" >> $GITHUB_ENV
+          echo "SCRIPTPATH=$GITHUB_WORKSPACE" >> $GITHUB_ENV
+          echo "PLUGIN_ABS_PATH=$GITHUB_WORKSPACE/.." >> $GITHUB_ENV
+          if [ "${{ matrix.cubit }}" = "17_1_0" ]; then
+            echo "CUBIT_PATH=/opt/Trelis-17.1" >> $GITHUB_ENV
+            echo "CMAKE_ADDITIONAL_FLAG=-DCMAKE_CXX_FLAGS=-D_GLIBCXX_USE_CXX11_ABI=0" >> $GITHUB_ENV
+          else
+            echo "CUBIT_PATH=/opt/Coreform-Cubit-2020.2" >> $GITHUB_ENV
+          fi
+
+
+      - name: Apt dependencies
+        shell: bash
+        run: |
+          sudo apt-get update -y
+          sudo apt-get install -y g++ libeigen3-dev libhdf5-dev patchelf git cmake
+       
+      - name: Build MOAB
+        shell: bash
+        run: |
+          cd ${PLUGIN_ABS_PATH}
+          mkdir -pv moab/bld
+          cd moab
+          git clone https://bitbucket.org/fathomteam/moab -b Version5.1.0
+          cd bld
+          cmake ../moab $CMAKE_ADDITIONAL_FLAG \
+                -DENABLE_HDF5=ON \
+                -DCMAKE_PREFIX_PATH=/usr/lib/x86_64-linux-gnu/hdf5/serial \
+                -DBUILD_SHARED_LIBS=ON \
+                -DENABLE_BLASLAPACK=OFF \
+                -DENABLE_FORTRAN=OFF \
+                -DCMAKE_INSTALL_PREFIX=${PLUGIN_ABS_PATH}/moab
+
+          make -j`grep -c processor /proc/cpuinfo`
+          make install
+          cd ../..
+          rm -rf moab/moab moab/bld
+
+      - name: build DAGMC
+        shell: bash
+        run: |
+          cd ${PLUGIN_ABS_PATH}
+          mkdir -pv DAGMC/bld
+          cd DAGMC
+          git clone https://github.com/svalinn/DAGMC -b develop
+          cd bld
+          cmake ../DAGMC $CMAKE_ADDITIONAL_FLAG \
+                -DMOAB_DIR=${PLUGIN_ABS_PATH}/moab \
+                -DBUILD_UWUW=ON \
+                -DBUILD_TALLY=OFF \
+                -DBUILD_BUILD_OBB=OFF \
+                -DBUILD_MAKE_WATERTIGHT=ON \
+                -DBUILD_SHARED_LIBS=ON \
+                -DBUILD_STATIC_LIBS=OFF \
+                -DBUILD_EXE=OFF \
+                -DCMAKE_POSITION_INDEPENDENT_CODE=ON \
+                -DCMAKE_BUILD_TYPE=Release \
+                -DCMAKE_INSTALL_PREFIX=${PLUGIN_ABS_PATH}/DAGMC
+          make -j`grep -c processor /proc/cpuinfo`
+          make install
+          cd ../..
+          rm -rf DAGMC/DAGMC DAGCM/bld
+
+      - name: Trelis setup
+        shell: bash
+        env:   
+          CUBIT_DEB_URL: ${{ secrets[matrix.URL] }}
+          CUBIT_SDK_URL: ${{ secrets[matrix.URL_SDK] }}
+        run: |
+          if [ "${{ matrix.cubit }}" = "2020_2" ]; then
+            sudo apt-get -y install libglew2.1 libglu1-mesa libopengl0 ocl-icd-libopencl1 libcxsparse3 libopenblas0 libegl1
```

Comment:
Do we know why these libraries are needed for `Cubit-Learn 2020.2` but not `Trelis 17.1`?

Reply:
I don't know why, just experience the dpkg complaining about those

Reply:
Just learning this today... if we use `apt` to install the Debian package the dependencies will be handled for us.

`sudo apt install ./Coreform-Cubit-2020.2-Ubuntu20.deb`